index.js

```diff
@@ -90,7 +114,7 @@ program
     }
   });
 
-if (process.stdin.isTTY) {
+if (process.stdin.isTTY || process.argv[2] === "x") {
```

Comment:
@yassine-cc does this mean this already worked?

Reply:
Yes